docs/user-guide/concepts.md

```diff
@@ -322,28 +322,46 @@ for policy purposes.*
 
 ### Exceptions and propagation control
 
-By default, HNC propagates _all_ objects of a [specified type](how-to.md#admin-resources)
-from ancestor namespaces to descendant namespaces. However, sometimes this is
-too restrictive, and you need to create ***exceptions*** to certain policies. For example:
+You can choose whether HNC propagates _all_ objects of a [specified type](how-to.md#admin-resources)
+from ancestor namespaces to descendant namespaces by using the `Propagate`
+[synchronization mode](how-to.md#modify-the-resources-propagated-by-hnc), or if HNC only
+propagates objects of a specified type when selectors are used, using the `AllowPropagate`
+synchronization mode.
+
+When `Propagate` [synchronization mode](how-to.md#modify-the-resources-propagated-by-hnc) is
+configured for a [specified type](how-to.md#admin-resources), HNC propagates _all_ objects of this type
+from ancestor namespaces to descendant namespaces. When `AllowPropagate` synchronization mode
+is configured for a specified type, HNC does not propagates _any_ objects of this type
+from ancestor namespaces to descendant namespaces. ***Exceptions*** are used to create
+certain policies to modify the behavior of the propagation. For example:
```

Comment:
This text goes live immediately so we don't want people using HNC v1.0 to think that this is available when it's not. Also, for at least one minor release, we'll call this "beta."

So how about this:
* Restore the original text, but add a new section (above the one called "Built-in exceptions") called "[Beta in v1.1] Inclusive propagation" and describe `AllowPropagate` there.
* Soften the intro line from "By default, HNC propagates _all_ objects" to "HNC typically propagates _all_ objects" which is becoming more and more accurate.